docs/getting-started/index.md

```diff
@@ -1 +1,129 @@
-# Getting Started Guide
\ No newline at end of file
+# Getting Started Guide
+
+Keyv provides a consistent interface for key-value storage across multiple backends via storage adapters. It supports TTL based expiry, making it suitable as a cache or a persistent key-value store. Follow the steps below to get you up and running.
+
+## 1. Make a Project Directory
+Make a directory with your project in it.
+
+```sh
+mkdir keyv
+cd keyv
+```
+You’re now inside your project’s directory.
+
+## 2. Install keyv
+
+```sh
+npm install --save keyv
+```
+By default, everything is stored in memory; you can optionally also install a storage adapter; choose one from the following:
+
+```sh
+npm install --save @keyv/redis
+npm install --save @keyv/mongo
+npm install --save @keyv/sqlite
+npm install --save @keyv/postgres
+npm install --save @keyv/mysql
+npm install --save @keyv/etcd
+```
+
```

Comment:
Would be good right here to provide the link to our other 3rd party storage adapters